src/features/users.ts

```diff
@@ -0,0 +1,29 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { getUsers } from '../api/users';
+import { User } from '../types/User';
+
+type UserState = {
+  users: User[];
+};
+
+const initialState: UserState = {
+  users: [],
+};
+
+export const fetchUsers = createAsyncThunk('users/fetch', () => {
+  return getUsers();
```

Comment:
Ensure that the `getUsers` function is correctly implemented and handles errors properly. This will help prevent issues when fetching users.